docs/index.rst

```diff
@@ -18,21 +9,12 @@ More Examples
   magics
   Official Google BigQuery Magics Tutorials <https://cloud.google.com/bigquery/docs/visualize-jupyter>
 
-API Reference
--------------
-
-.. toctree::
-  :maxdepth: 2
-
-  reference
-  dbapi
-  design/index
 
 Migration Guide
 ---------------
 
-See the guides below for instructions on migrating from older to newer *major* releases
-of this library (from ``1.x`` to ``2.x``, or from ``2.x`` to ``3.x``).
+Migrating from the ``google-cloud-bigquery``, you need to run the ``%load_ext bigquery-magics``
```

Comment:
This reads strangely. Please show a "before" and "after" code sample.

Reply:
Done.